features/media-queries-update.yml

```diff
@@ -0,0 +1,6 @@
+name: Update frequency media feature
```

Comment:
I was unsure about name here- the key is `update` but I think `Update frequency" is more useful as a description.